modules/core/src/main/java/org/locationtech/jts/geomgraph/Edge.java

```diff
@@ -178,9 +178,8 @@ public void addIntersection(LineIntersector li, int segmentIndex, int geomIndex,
       /**
       * Add the intersection point to edge intersection list.
       */
-      EdgeIntersection ei = eiList.add(intPt, normalizedSegmentIndex, dist);
```

Comment:
Don't see need to remove this

Reply:
Doh!  That's not what I meant to do.  Only the ei local var is unused.  I presume the add critical.  I'll push a fix in a bit.  And I could use guidance on how you'd like the style.  e.g. I'm mixing `/* */` and `//` here.  That ok?  Just use `/* */` or remove them all?

```java
      /* EdgeIntersection ei = */ eiList.add(intPt, normalizedSegmentIndex, dist);                                                                               
      // ei.print(System.out);                                                                                                                                   
```